gateway/ln-gateway/src/bin/cln_extension.rs

```diff
@@ -197,7 +197,11 @@ impl ClnRpcService {
             ))
         } else {
             Err(ClnExtensionError::Error(anyhow!(
-                "Failed to start cln plugin"
+                "
+                    Failed to start cln plugin - another instance of lightningd may already be running.
```

Comment:
Possibly this should be just logging message

`error!(Failed to start cln plugin - another instance of lightningd may already be running.);`

?

Reply:
Noted

Reply:
I think you still need to return an `Err` from this function though, since it is expecting a return value